### DISCUSSION TITLE
Arduino UNO R4 Minima with ST7735 TFT display

### DISCUSSION THREAD

**Discussion by StephenMNolan** [moononournation/Arduino_GFX#342] [Q&A] [answered]
Not working for me. Any suggestions would be appreciated.
Code follows:
/*
  Test TFT display on Arduino UNO R4 Minima.
*/
#include <Arduino_GFX_Library.h> // Library for TFT display

/* Set up the display pins
D8 - RST output to the display RST input 
D9 - DC output to the display RS input
CS/D10 - CS output to the display CS input
COPI/D12 – Serial data output to the display SDA input
SCK/D13 – Clock output to the display CLK input */

// Using HW SPI pins
Arduino_DataBus *bus = new Arduino_HWSPI(/*DC=*/9 , /*CS=*/10);
Arduino_GFX *gfx = new Arduino_ST7735(bus, /*RST=*/8 );

void setup(void) // Required setup routine
{
  gfx->begin();
  gfx->fillScreen(BLACK);
  gfx->setCursor(10, 10);
  gfx->setTextColor(RED);
  gfx->println("Hello World!");
}

void loop(void) // Required main loop
{
  while(true);{} // Sit in a loop forever  
}

**Comment by moononournation** [comment]
I did not have R4 in hand, so it is not tested yet.

**Comment by StephenMNolan** [comment]
Well I just tested the exact same code on a Micro (with appropriate pin changes) and it works so this library apparently does not work with Minima.

**Comment by ZinggJM** [comment, marked as answer]
There happened a misunderstanding.
My fork https://github.com/ZinggJM/Arduino_GFX was added to the list https://github.com/arduino/uno-r4-library-compatibility#readme, but it just addresses use of MCUFriend UNO 8bit shields on Arduino UNO R4.
See https://forum.arduino.cc/t/support-mcufriend-uno-tft-shield-on-uno-r4/1151857

It is not yet synched with this repo, as I have not yet received the Arduino UNO R4 minima to test with.
I plan to provide a pull request afterwards.
Jean-Marc Zingg
